mozilla-release/build

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -x # Show commands
+set -eu # Errors/undefined vars are fatal
+set -o pipefail # Check all commands in a pipeline
```

Comment:
So we don't want to get any indexing for these for now? It sounds fine, just maybe worth noting in a comment?

Reply:
Added comments in all the new `build` files. I do plan on adding Rust/C++ indexing for mozilla-beta and mozilla-release once bug 1533801 merges to those branches and we can pull the artifacts from taskcluster.